src/api/app/jobs/issue_tracker_fetch_issues_job.rb

```diff
@@ -2,6 +2,6 @@ class IssueTrackerFetchIssuesJob < ApplicationJob
   queue_as :issuetracking
 
   def perform(issue_tracker_id)
-    IssueTracker.find(issue_tracker_id).fetch_issues
+    IssueTracker.find_by_id(issue_tracker_id).try(:fetch_issues)
```

Comment:
`find_by_id` is equivalent to `find`. Please do not use the dynamic version. :see_no_evil: